src/main/java/frc/robot/subsystems/intake/commands/StartIntake.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.subsystems.intake.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.intake.Intake;
+
+public class StartIntake extends CommandBase {

```

Comment:
add `execute` function

Reply:
adding it would be unnecessary

Reply:
if the purpose of the class is to "power the wheels", then you need to use `intake.setVelocity(Constants.Intake.VELOCITY);` in the `execute` function